astropy_helpers/extern/automodapi/automodapi.py

```diff
@@ -1,13 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
-This sphinx extension adds a tools to simplify generating the API
-documentation for Astropy packages and affiliated packages.
-
-.. _automodapi:
-
-========================
-automodapi directive
```

Comment:
The headings are no longer needed in the generated docs: http://sphinx-automodapi.readthedocs.io/en/latest/automodapi.html